.travis.yml

```diff
@@ -8,18 +8,16 @@ cache:
   directories:
     - bin/
 
+dist: bionic
```

Comment:
What does this do?

Reply:
Changes the version of ubuntu used by travis. Bionic is the most recent LTS. It was needed for xvfb, but we don't need xvfb now...